05.py

```diff
@@ -0,0 +1,16 @@
+# 소득 불균형
+
+T= int(input())
+for t in range(1,T+1):
+    p = int(input())
+    money = list(map(int,input().split()))
+    
+    avg = sum(money) // p
```

Comment:
깔끔하네요! 👍 
```suggestion
    avg = sum(money) / p
```
추가로 `//` 연산은 몫을 구하는 연산으로, 평균을 구하려면 `/` 연산을 활용해주시는 것이 맞습니다!